mautic/mautic
# Add border radius to button slot type

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Not always rounded buttons are  cool :)
Also fix bug with styling all A elements

#### Steps to test this PR:
1.  Apply PR  and call command php app/console mautic:aassets:generate
2. Add button to template  and try change border radius
3. Change background color and check if close button didn't change background color
